hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -72,6 +72,7 @@ class SubmitRequestBodyBase(DictParseable):
     content_id: str
     content_type: t.Type[ContentType]
     additional_fields: t.Optional[t.List]
+    resubmit: bool = False
```

Comment:
nit/personal-pref: do you want to make this more explicit like `force_resubmit`?